src/components/Select/Select.tsx

```diff
@@ -1,15 +1,22 @@
 import * as React from 'react';
 import classnames from 'classnames';
-
 import { px, useNormalizedInputProps } from '../../utils';
-
 import { InputProps } from '../Input/Input';
+import { SelectVariants } from './types';
 
 export interface SelectProps extends InputProps {
   /**
    * Option elements that are selectable
    */
   children: React.ReactNode;
+  /**
+   * Determines if you want to show the icon
+   */
+  showCustomIcon?: boolean;
```

Comment:
NBD but naming is somewhat confusing, since you can't pass in your own custom icon. I was almost expecting another prop called `customIcon` which let me pass in my own icon component

could be something like, `showArrowIcon` or `showCaretIcon` or `showIcon` or something like that

Reply:
IMHO the custom Icon should always render in place of the default Select appearance as the Easel compendium has it there @constantinehuzenko  and the default HTML `appearance: none` should be used for that Select component